client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/ConfigurationProperties.java

```diff
@@ -70,6 +70,7 @@
    public static final String BATCH_SIZE = "infinispan.client.hotrod.batch_size";
    public static final String TRANSACTION_MANAGER_LOOKUP = "infinispan.client.hotrod.transaction.transaction_manager_lookup";
    public static final String TRANSACTION_MODE = "infinispan.client.hotrod.transaction.transaction_mode";
+   public static final String SINGLE_PORT = "infinispan.client.hotrod.single_port";
```

Comment:
Please update `ConfigurationTest` to verify this is read/written correctly.

Reply:
Done